### PR TITLE
Community PHP implementation

### PR DESCRIPTION
**Pull request by whitemerry**

### CODE REVIEW

**Comment by adriancole** [comment]
Thanks for the this! I added some notes. Minimally, I think the tracer
should work with zipkin server via some means before it is added to the
list.

File: _data/community_instrumentations.yml [hunk 1]
```diff
@@ -146,3 +146,12 @@
   transports: Http
   sampling: "Yes"
   notes: Uses Brave4; Play 2.3, 2.4 and 2.5 support.
+
+- language: PHP
+  library: >-
+    [phpkin](https://github.com/whitemerry/phpkin)
+  framework: Any
+  propagation: User implementation (B3 Storage provided)
+  transports: Manual bind to existing clients
```
Comment (adriancole) [inline _data/community_instrumentations.yml]: maybe change this to "http, log file"

File: _data/community_instrumentations.yml [hunk 2]
```diff
@@ -146,3 +146,12 @@
   transports: Http
   sampling: "Yes"
   notes: Uses Brave4; Play 2.3, 2.4 and 2.5 support.
+
+- language: PHP
+  library: >-
+    [phpkin](https://github.com/whitemerry/phpkin)
+  framework: Any
+  propagation: User implementation (B3 Storage provided)
```
Comment (adriancole) [inline _data/community_instrumentations.yml]: Maybe "B3, custom"

Comment (whitemerry) [inline _data/community_instrumentations.yml]: updated

**Comment by adriancole** [comment]
thanks much!

**Pull request merged**